game-core/src/main/java/games/strategy/triplea/delegate/battle/steps/BattleSteps.java

```diff
@@ -58,7 +59,10 @@
         new ClearFirstStrikeCasualties(battleState, battleActions);
     final BattleStep offensiveStandard = new OffensiveGeneral(battleState, battleActions);
     final BattleStep defensiveStandard = new DefensiveGeneral(battleState, battleActions);
-    final BattleStep generalRetreat = new OffensiveGeneralRetreat(battleState, battleActions);
+    final BattleStep offensiveGeneralRetreat =
+        new OffensiveGeneralRetreat(battleState, battleActions);
+    final BattleStep clearGeneralCasualties =
```

Comment:
Any thoughts to rename 'clear' to 'remove'? They are mostly synonymous, IMO 'remove' is just slightly more descriptive.

Reply:
Good idea except we already have `ClearAaCasualties` and `ClearFirstStrikeCasualties` which do the same thing for the other battle phases.  I'd like to keep their names similar.  And since they are stored in the save game, we would have to either just duplicate the class or do some sort of deserialization magic to handle the rename.

Reply:
Fair enough :+1: